Containers/mastercontainer/start.sh

```diff
@@ -200,6 +200,11 @@ It needs to be a string with letters, numbers, hyphens and underscores.
 It is set to '$APACHE_ADDITIONAL_NETWORK'."
         exit 1
     fi
+    tmp_out=$(sudo -u www-data docker network connect $APACHE_ADDITIONAL_NETWORK nextcloud-aio-mastercontainer 2>&1)
+    if [ -n "$tmp_out" ] && [[ ! "$tmp_out" =~ "nextcloud-aio-mastercontainer already exists in network $APACHE_ADDITIONAL_NETWORK" ]]; then
+        print_red "Unable to connect to $APACHE_ADDITIONAL_NETWORK. Cannot continue. Error: \n $tmp_out"
+        exit 1
+    fi
```

Comment:
```suggestion
    tmp_out=$(sudo -u www-data docker network connect $APACHE_ADDITIONAL_NETWORK nextcloud-aio-mastercontainer 2>&1)
    # not tested
    if [ $? -ne 0 ]; then
        print_red "Unable to connect to $APACHE_ADDITIONAL_NETWORK. Cannot continue. Error: \n $tmp_out"
        exit 1
    fi
```
Is it possible to use the exit code? that would be more relevant

Reply:
Originally I did use exit code in `$?`, but that is also non-zero when `nextcloud-aio-mastercontainer` is already connected to the specified network; which can happen because someone explicitly connected them via the docker compose file or because it's being restarted etc.
That's why I rewrote it to above code which catches the output and excludes the condition `nextcloud-aio-mastercontainer already exists in network`

Reply:
I can potentially change it to this, if that's better?
```
    if [ $? -ne 0 ] && [[ ! "$tmp_out" =~ "nextcloud-aio-mastercontainer already exists in network $APACHE_ADDITIONAL_NETWORK" ]]; then
```

Reply:
Does the exit code change depending on the error type?

Reply:
No, it does not. It's always 1.